docs/dma/dma-configurationsettings.md

```diff
@@ -137,6 +137,17 @@ for source and target instances while running an assessment or migration, by set
 </appSettings>
 ```
 
+## Ignore error codes
+
+Each rule has an error code in its title. You might not care about the specific rules and wants to ignore them. You can control the ignoreErrorCodes property. You can ignore one error or multiple errors that are separated by semi comma. For example, ignoreErrorCodes="46010;71501".The default value is 71501. 71501 means unresolved references found when the object references system objects like procedures, views, etc.

```

Comment:
This should read as follows:

Each rule has an error code in its title. If you don't need rules and want to ignore them, use the **ignoreErrorCodes** property. You can specify to ignore a single error or multiple errors. To ignore multiple errors, use a semicolon, e.g., ignoreErrorCodes="46010;71501". The default value is 71501, which is associated with unresolved references identified when an object references system objects such as procedures, views, etc.

Reply:
resolved